test/attributes.test.ts

```diff
@@ -133,7 +132,7 @@ describe("Attributes", () => {
         expect(notificationResponse.isSuccess).toBe(true);
 
         await syncUntilHasMessageWithNotification(client2, notificationResponse.result.notificationId);
-        await client2._eventBus!.waitForEvent<DataEvent<any>>("consumption.peerSharedAttributeSucceeded", (event: DataEvent<SuccessionEventData>) => {
+        await client2._eventBus!.waitForEvent<DataEvent<any>>("consumption.peerSharedAttributeSucceeded", (event: DataEvent<any>) => {
```

Comment:
Why were the event types removed in this file?

Reply:
runtime was used in connector tests, major flaw.